src/agents/co_writer/narrator_agent.py

```
@@ -128,7 +128,9 @@ def _validate_tts_config(self):
         self.logger.info(f"  API Key: {api_key_preview}")
         self.logger.info(f"  Default Voice: {self.default_voice}")
 
-    async def process(self, content: str, style: str = "friendly", voice: str = None, skip_audio: bool = False) -> dict[str, Any]:
+    async def process(
+        self, content: str, style: str = "friendly", voice: str = None, skip_audio: bool = False
```

Ruff (0.14.10)

PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)